source/_components/sensor.azure_maps_travel_time.markdown

```diff
@@ -0,0 +1,145 @@
+---
+layout: page
+title: "Azure Maps Travel Time"
+description: "Instructions on how to add Azure Maps travel time to Home Assistant."
+date: 2018-11-20 18:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: azure_maps.png
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.83
+---
+
+The `azure_maps_travel_time` sensor provides travel time from the [Azure Maps Route Directions API](https://docs.microsoft.com/en-us/rest/api/maps/route/getroutedirections).
+
+## {% linkable_title Setup %}
+
+You need to register for an API key by following the instructions [here](https://docs.microsoft.com/en-us/azure/azure-maps/how-to-manage-account-keys).
```

Comment:
There are two keys - primary and secondary. Can you clarify which one to be used, if it matters?